subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_security.c

```diff
@@ -29,6 +29,45 @@ LOG_MODULE_REGISTER(lwm2m_security, CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL);
 #define SERVER_SHORT_SERVER_ID 0
 #define SERVER_LIFETIME_ID 1
 
+static struct modem_mode_change mm;
+
+static struct  modem_conn {
+	int  rtime;    /* Time to next connection attempt */
+	bool bconnect; /* Block on connect */
+} conn_retry;
```

Comment:
These values are not used anymore.

We only do this in blocking mode, and rtime has no use for us.